CONTRIBUTING.md

```diff
@@ -132,7 +132,7 @@ You will need basic `git` proficiency to contribute to
 manual. Type `git --help` in a shell and enjoy! If you prefer books, [Pro
 Git](https://git-scm.com/book/en/v2) is a very good reference.
 
-You'll need **[Python 3.8](https://github.com/huggingface/transformers/blob/main/setup.py#L426)** or above to contribute to 🤗 Transformers. Follow the steps below to start contributing:
+You'll need **[Python 3.8](https://github.com/huggingface/transformers/blob/4c040aba02b0283619a06bdc40ecf868508b9e52/setup.py#L449)** or above to contribute to 🤗 Transformers. Follow the steps below to start contributing:
```

Comment:
In this case, I think we should keep pointing to the `main` branch as this is the up-to-date version spec

```suggestion
You'll need **[Python 3.8](https://github.com/huggingface/transformers/blob/main/setup.py#L449)** or above to contribute to 🤗 Transformers. Follow the steps below to start contributing:
```

There aren't lots of new libraries added to setup.py so should be fairly stable